clap-v3-utils/src/keypair.rs

```diff
@@ -371,148 +367,6 @@ impl DefaultSigner {
     }
 }
 
-#[derive(Debug, Clone)]
```

Comment:
moving all of these public symbols without leaving re-exports will break all consumers

Reply:
Sorry if I am misunderstanding this, but Is this the case even if `SignerSource` is currently restricted within the `clap-v3-utils` crate?

Reply:
ah i missed that they were both moved _and_ promoted to public simultaneously. generally prefer not to do cosmetic changes like reorging private symbols in the same change set as functionals